README.md

```diff
@@ -1,3 +1,20 @@
+# Actions - GitHub Actions for Azure
+
+This repository provides a framework, guidleines and processes to author new and contribute to existing GitHub Actions deploying to Azure.
+
+## Table of contents
+- [GitHub Actions for deploying to Azure](#github-actions-for-deploying-to-azure)
+- [GitHub Actions for Azure](#github-actions-for-azure)

```

Comment:
This feels like a repetition of first line item

Reply:
Changed